src/main/java/net/atos/zac/notificaties/NotificatieReceiver.java

```diff
@@ -59,7 +59,7 @@ public class NotificatieReceiver {
     private ConfiguratieService configuratieService;
 
     @Inject
-    private ZoekenService zoekenService;
+    private IndexeerService indexeerService;
 
     @POST
     public Response notificatieReceive(final Notificatie notificatie) {
```

Comment:
Volgens mij is dit niet genoeg. Er zijn ook nog andere notificaties die je moet handlen. In elk geval ZAKEN.STATUS... ZAKEN.ROL... ZAKEN.RESULTAAT... zou ik zeggen.